packages/client/src/components/SafeSettings/components/Owners/AddSafeOwner.js

```diff
@@ -36,50 +46,57 @@ const AddSafeOwner = ({ onCancel, onNext, safeOwners }) => {
   };
 
   const nextButtonClasses = [
-    "button flex-1 is-primary",
-    addressValid ? "" : "disabled",
+    "button flex-1 is-primary ",
+    addressValid ? "has-text-weight-bold" : "disabled",
   ];
+  const addrInputClasses = [
+    "p-4 rounded-sm column is-full is-size-6",
+    addressBorderClass
+  ]
 
   return (
     <>
       <div className="p-5">
-        <h2 className="is-size-4 has-text-black">Add a new safe owner</h2>
-        <p className="has-text-grey">
-          Heads up, they'll have full access to this safe
-        </p>
+        <h2 className="is-size-4 has-text-black">Add Owner</h2>
       </div>
-      <div className="border-light-top p-5 has-text-grey">
-        <div className="flex-1 is-flex is-flex-direction-column">
-          <label className="has-text-grey mb-2">Owner Name</label>
-          <input
-            className="p-4 rounded-sm border-light"
-            type="text"
-            placeholder="Enter local owner name"
-            value={name}
-            onChange={(e) => setName(e.target.value)}
-          />
-        </div>
-        <div className="flex-1 is-flex is-flex-direction-column mt-4">
-          <label className="has-text-grey mb-2">
-            Address<span className="has-text-red">*</span>
+      <div className="border-light-top border-light-bottom p-5 has-text-grey">
+        <div className="flex-1 is-flex is-flex-direction-column mb-4">
+          <label className="has-text-grey mb-2 with-icon">
+            Address<span className="has-text-red">*</span> <Svg name="QuestionMark" style={{position: "relative", top: 4 }} />
```

Comment:
```suggestion
            Address<span className="has-text-red">*</span> <Svg name="QuestionMark" className="ml-1" />
```

Reply:
this doesnt seem to address the vertical alignment issue